backend/internal/model/game_test.go

```diff
@@ -197,6 +204,9 @@ func TestUpdateShotsData(t *testing.T) {
 
 			assert.Equal(t, tt.expectedCountWhite, game.gameData.WhiteShotsCount)
 			assert.Equal(t, tt.expectedCountBlue, game.gameData.BlueShotsCount)
+			assert.Equal(t, tt.expectedWhiteAtGoalCount, game.gameData.WhiteAtGoalCount)
+			assert.Equal(t, tt.expectedBlueAtGoalCount, game.gameData.BlueAtGoalCount)
```

Comment:
Maybe if you would create one structure expectedGameData with type GameStats, you could use only one assert.Equal() and compare it to game.gameData structure.

Reply:
Done